scripts/tiqit/__init__.py

```diff
@@ -998,7 +998,7 @@ def getBasePath():
     return basepath
 
 def getBaseHost():
-    return 'http://%s%s' % (os.environ['HTTP_HOST'], getBasePath())
+    return 'https://%s%s' % (os.environ['HTTP_HOST'], getBasePath())
```

Comment:
Should doc/examples/apache_config be updated to use the HTTPS port number instead of (or as well as) the HTTP port number?

Reply:
Moved to use https only (replaced the http port num)